package/python-kiwi-spec-template

```diff
@@ -531,14 +417,6 @@ done
     --install %_bindir/kiwicompat kiwicompat %_bindir/kiwicompat-3 10
 %endif
```

Comment:
Can we rip alternatives out entirely now? We don't need it if there's only one variant, right?

Reply:
yes true, haven't thought about that part. How would we handle if different versions of py3 are installed ? py3.6, py3.7 ? suse doesn't support this, but on Fedora for example you can have multiple python3's installed

Reply:
One of the reasons why the gitlab pipeline uses a fedora container that comes with many py3 versions pre-installed

Reply:
Yes, but packages cannot depend on anything except the main python3 environment (which is the `python3` package). All the other ones are for dev/test only.

Reply:
ok in this case we can rip out alternatives completely... that's nice

Reply:
you did it :)